kusto/ingest/internal/conn/conn.go

```diff
@@ -51,15 +51,18 @@ func New(endpoint string, auth kusto.Authorization, client *http.Client) (*Conn,
 		).SetNoRetry()
 	}
 
-	return newWithoutValidation(endpoint, auth, client)
+	return newWithoutValidation(endpoint, auth, client, application, user, version)
 }
 
-func newWithoutValidation(endpoint string, auth kusto.Authorization, client *http.Client) (*Conn, error) {
+func newWithoutValidation(endpoint string, auth kusto.Authorization, client *http.Client, application, user, version string) (*Conn, error) {
```

Comment:
Why is this a duplicate of Kusto/conn.go
Also, for streamingingestion you can pass CRP so this should look similar to the other conn.

Reply:
conn.go is a private type, so we need to duplicate it here in this package.
Alternatively we can have this functionality be a part of the regular "conn" under an interface and reduce the duplication maybe